README.md

```diff
@@ -225,6 +225,9 @@ it is recommended not to preinstall containerd in MachineSet or BYOH Windows ins
 Windows Nodes do not support workloads created via DeploymentConfigs. Please use a normal Deployment, or other method to
 deploy workloads.
 
+### Cluster-wide proxy 
+WMCO does not support adding Windows workloads using a [cluster-wide proxy](https://docs.openshift.com/container-platform/latest/networking/enable-cluster-wide-proxy.html) config for the OpenShift Container Platform. WMCO will not be able to automatically route proxy connections for Windows workloads.
```

Comment:
The Windows Machine Config Operator (WMCO) does not support adding Windows workloads using a cluster-wide proxy configuration file. The WMCO is not able to automatically route proxy connections for Windows workloads.